### PR TITLE
feat: LIQF-fees in growing market feature tests

### PR DESCRIPTION
**Pull request by cdummett**
### Description
New feature test with following scenarios:
- Scenario: 001 2 LPs joining at start, unequal commitments. Checking calculation of equity-like-shares and liquidity-fee-distribution in a market with small growth (0042-LIQF-008 0042-LIQF-011)
- Scenario: 002 2 LPs joining at start, unequal commitments. Checking calculation of equity-like-shares and liquidity-fee-distribution in a market with large growth (0042-LIQF-008 0042-LIQF-011)

Issues calculating EPS as defined following spec [0042](https://github.com/vegaprotocol/specs/blob/master/protocol/0042-LIQF-setting_fees_and_rewarding_lps.md).

### CODE REVIEW

File: core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature [hunk 1]
```diff
@@ -0,0 +1,930 @@
...
+
+    # Confirm equity-like-shares updated immediately after liquidity amendment
+    Then the liquidity provider fee shares for the market "ETH/MAR22" should be:
+      | party | equity like share    | average entry valuation           |
+      | lp1   | 0.0794314813996635   | 3971.5740699831772119860611436306 |
+      | lp2   | 0.9205685186003365   | 46000                             |
+
+    # ERROR:
+    # - "equity-like-share" values calculated incorrectly.
+    # - "average-entry-valuation" values calculated incorrectly, sum of "average-entry-valuation" values does not match sum of "physical-stake" values.
```
Comment (EVODelavega) [inline core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature:241]: I don't see how we'd expect the sum of the average entry valuation to match the sum of the physical stake

Comment (cdummett) [reply core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature:241]: Hey, yep following the updated `average-entry-valuation` in spec 0042 those comments in the test don't make sense any more. I'll remove the comments from the feature test and update all the values.

**Review by Jiajia-Cui — DISMISSED** [review]
LGTM

File: core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature [hunk 2]
```diff
@@ -0,0 +1,1155 @@
...
+
+    # ERROR:
+    # - "equity-like-share" values calculated incorrectly.
+    # - "average-entry-valuation" values calculated incorrectly.
+
+    # EXPECTED:
+    # Then the liquidity provider fee shares for the market "ETH/MAR22" should be:
+    #  | party | equity like share | average entry valuation |
+    #  | lp1   | 0.079809          | 15628.748               |
+    #  | lp2   | 0.920191          | 50000.000               |
```
Comment (Jiajia-Cui) [inline core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature:247]: please tidy up the comments on the expected value on average entry valuation as they are different from the value in line 236

File: core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature [hunk 3]
```diff
@@ -0,0 +1,1155 @@
...
+      | lp2   | 0.9205685186003365   | 50000                             |
+
+    # ERROR:
+    # - "equity-like-share" values calculated incorrectly.
+    # - "average-entry-valuation" values calculated incorrectly.
+
+    # EXPECTED:
+    # Then the liquidity provider fee shares for the market "ETH/MAR22" should be:
+    #  | party | equity like share | average entry valuation |
+    #  | lp1   | 0.079809          | 15628.748               |
```
Comment (Jiajia-Cui) [inline core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature:296]: please tidy up the comments

File: core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature (hunk 2 above)
Comment (cdummett) [reply core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature:247]: Hey, the values are different from in line 236 as there is an error still.

Values in the "EXPECTED:" section have been updated to match values from the [sheet](https://docs.google.com/spreadsheets/d/1ywOsc9QivCzyKE0TNgflD9EQjZ9cXGiSTiW3efEy22w/edit#gid=1778207922) after the spec change.

File: core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature (hunk 3 above)
Comment (cdummett) [reply core/integration/features/verified/0042-LIQF-fees_rewards_growing_market.feature:296]: Same as comment above ^^^

**Review by EVODelavega — APPROVED** [review]
LGTM

**Review by Jiajia-Cui — APPROVED** [review]
LGTM

**Pull request merged**